diff --git a/PKGBUILD b/PKGBUILD.new
index 967d14dd7..bbc8c662d 100644q
--- a/PKGBUILD
+++ b/PKGBUILD.new
@@ -29,6 +29,12 @@ prepare() {
   rsync -ar --mkpath --info=progress2 --no-inc-recursive $SRCPATH/ $SRCPATH/src/$_srcname
 }
 
+build() {
+  cd $SRCPATH/src/$_srcname
+  make -j$CORECOUNT
+  make -j$CORECOUNT modules
+}
+
 _package() {
   pkgdesc="The kernel and modules"
   depends=(coreutils kmod initramfs)
@@ -138,30 +144,7 @@ _package-headers() {
   chmod -Rc 644 "$pkgdir"
 }
 
-_package-docs() {
-  pkgdesc="Documentation for the $pkgdesc kernel"
-
-  cd $_srcname
-  local builddir="$pkgdir/usr/lib/modules/$(<version)/build"
-
-  echo "Installing documentation..."
-  local src dst
-  while read -rd '' src; do
-    dst="${src#Documentation/}"
-    dst="$builddir/Documentation/${dst#output/}"
-    install -Dm644 "$src" "$dst"
-  done < <(find Documentation -name '.*' -prune -o ! -type d -print0)
-
-  echo "Adding symlink..."
-  mkdir -p "$pkgdir/usr/share/doc"
-  ln -sr "$builddir/Documentation" "$pkgdir/usr/share/doc/$pkgbase"
-
-  echo "Fixing permissions..."
-  chmod -Rc 644 "$pkgdir"
-  rm -rf $SRCDIR/pkg $SRCDIR/src
-}
-
-pkgname=("$pkgbase" "$pkgbase-headers" "$pkgbase-docs")
+pkgname=("$pkgbase" "$pkgbase-headers")
 for _p in "${pkgname[@]}"; do
   eval "package_$_p() {
     $(declare -f "_package${_p#$pkgbase}")
